opensearch-operator/pkg/builders/cluster.go

```diff
@@ -233,7 +233,20 @@ func NewSTSForNodePool(
 	if len(cr.Spec.General.Command) > 0 {
 		startUpCommand = cr.Spec.General.Command
 	}
-	mainCommand := helpers.BuildMainCommand("./bin/opensearch-plugin", cr.Spec.General.PluginsList, true, startUpCommand)
+
+	var pluginslist []string
+	if cr.Spec.General.Monitoring.Enable {
+		if cr.Spec.General.Monitoring.PluginURL == "" {
+			pluginslist = append(pluginslist, fmt.Sprintf("https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/releases/download/%s.0/prometheus-exporter-%s.0.zip", cr.Spec.General.Version, cr.Spec.General.Version))
```

Comment:
The string should be extracted as a constant

Reply:
Moved out